diff --git a/node_modules/metro/src/lib/reporting.js b/node_modules/metro/src/lib/reporting.js
index b0d7b91..78c13ea 100644
--- a/node_modules/metro/src/lib/reporting.js
+++ b/node_modules/metro/src/lib/reporting.js
@@ -1,7 +1,12 @@
 "use strict";
 
 const chalk = require("chalk");
-const stripAnsi = require("strip-ansi");
+
+let stripAnsi;
+import('strip-ansi').then(mod => {
+    stripAnsi = mod.default || mod;
+});
+
 const util = require("util");
 function logWarning(terminal, format, ...args) {
   const str = util.format(format, ...args);
