diff --git a/node_modules/metro/src/lib/reporting.js b/node_modules/metro/src/lib/reporting.js
index b0d7b91..b0eb8ce 100644
--- a/node_modules/metro/src/lib/reporting.js
+++ b/node_modules/metro/src/lib/reporting.js
@@ -1,7 +1,7 @@
 "use strict";
 
 const chalk = require("chalk");
-const stripAnsi = require("strip-ansi");
+const stripAnsi = await import('strip-ansi').then(mod => mod.default || mod);
 const util = require("util");
 function logWarning(terminal, format, ...args) {
   const str = util.format(format, ...args);
